lib/src/services/files.dart

```diff
@@ -41,13 +45,13 @@ class FilesService extends Service {
   /// Map the command to file the output should into 
   /// 
   /// Used for logging rover metrics
-  Map<Message, File> modes = { ///name better
-    ArmData(): File(""),
-    GripperData(): File(""),
+  Map<String, File> modes = { ///name better
+    "ArmData": File(""),
+    "GripperData": File(""),
     //GpsCoordinates(): File(""),
-    ElectricalData(): File(""),
-    AutonomyData(): File(""),
-    DriveData(): File(""),
+    "ElectricalData": File(""),
+    "AutonomyData": File(""),
+    "DriveData": File(""),
     // MarsData()
   };
```

Comment:
Actually I think we can get rid of this entirely: 
- initialize the `output/logs/timestamp` folder in `init`
- save to `${data.messageName}.log` in `logData`
So you can end up with a file like `C:\users\levi\Documents\Dashboard\logs\2023-4-28-2-42\ScienceData.log`